cmd/run/run.go

```diff
@@ -249,6 +249,8 @@ func NewRunCommand() *cobra.Command {
 
 	flags.Duration("request-timeout", defaultConfig.RequestTimeout, "configures request timeout.  If both HTTP upstream timeout and request timeout are specified, request timeout will be used.")
 
+	flags.Bool("check-tracker-enabled", defaultConfig.CheckTrackerEnabled, `enabled/disabled tracking tuples`)
```

Comment:
Same as before, please improve this description

Reply:
👍